### PR TITLE
Disable Modifier for plane which has video layer

### PR DESCRIPTION
**Pull request by Shao-Feng**
[DO NOT MERGE] Just checking Linux compiling

### CODE REVIEW

File: common/display/displayplanemanager.cpp [hunk 1]
```diff
@@ -800,7 +800,7 @@ void DisplayPlaneManager::EnsureOffScreenTarget(DisplayPlaneState &plane) {
   }
 
   uint64_t modifier = plane.GetDisplayPlane()->GetPreferredFormatModifier();
-  if (plane.IsVideoPlane())
+  if (plane.HasVideoLayer())
```
Comment (js0701) [inline common/display/displayplanemanager.cpp]: If a plane has video layer, then it should be a video plane when going to VPP
Have you debug whey plane .IsVideoPlane is not true?

Comment (js0701) [reply common/display/displayplanemanager.cpp]: We called SetVideoPlane(true) when ForceVPP is called

Comment (Shao-Feng) [reply common/display/displayplanemanager.cpp]: Not clear yet, but "SetVideoPlane" only work for extend display.

Comment (js0701) [reply common/display/displayplanemanager.cpp]: Then we can fix ResetLayers logic,
without checking the layer count to be 1

**Pull request closed without merging**